pyClient/commands/zeth_ls_commits.py

```diff
@@ -15,12 +14,7 @@ def ls_commits(ctx: Any) -> None:
     """
     List all commitments in the joinsplit contract
     """
-    web3 = open_web3_from_ctx(ctx)
-    mixer_desc = load_mixer_description_from_ctx(ctx)
-    zeth_instance = mixer_desc.mixer.instantiate(web3)
-    null = bytes(32)
+    merkle_tree = open_merkle_tree(ctx)
```

Comment:
`open_merkle_tree` should receive a `Context`, so let's switch to  `Context` instead of `Any` in the signature of the `ls_commit` function

Reply:
Used `Context` everywhere.